--- vino-3.6.2/./server/vino-server.desktop.in.in~	2011-09-07 21:20:42.000000000 +0300
+++ vino-3.6.2/./server/vino-server.desktop.in.in	2013-01-16 21:58:35.511074418 +0200
@@ -3,7 +3,7 @@
 _Comment=GNOME Desktop Sharing Server
 Exec=@LIBEXECDIR@/vino-server
 Icon=preferences-desktop-remote-desktop
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;MATE;Unity;
 Terminal=false
 Type=Application
 AutostartCondition=GSettings org.gnome.Vino enabled
