diff -urN vino-2.7.91.orig/configure.in vino-2.7.91/configure.in
--- vino-2.7.91.orig/configure.in	2004-08-17 19:34:51.049690752 +0200
+++ vino-2.7.91/configure.in	2004-08-17 19:35:02.103010392 +0200
@@ -20,7 +20,7 @@
 AC_PATH_PROG(GCONFTOOL, gconftool-2)
 AC_PATH_PROG(GLIB_GENMARSHAL, glib-genmarshal)
 
-ALL_LINGUAS="bg bn cs en_CA en_GB es gu hu nl no pa pl pt pt_BR sq sr sr@Latn sv"
+ALL_LINGUAS="bg bn cs en_CA en_GB es gu hu nb nl pa pl pt pt_BR sq sr sr@Latn sv"
 AM_GLIB_GNU_GETTEXT
 
 GNOME_COMPILE_WARNINGS(yes)
