Index: trunk/configure.in
===================================================================
--- trunk/configure.in	(revision 876)
+++ trunk/configure.in	(working copy)
@@ -47,7 +47,7 @@
 	
 PKG_CHECK_MODULES(VINO_CAPPLET, glib-2.0 >= 2.12.0 gtk+-2.0 >= 2.10.0 gconf-2.0 libglade-2.0 libgnomeui-2.0 dbus-glib-1)
 
-PKG_CHECK_MODULES(VINO_TOOLS, glib-2.0 >= 2.12.0 gconf-2.0)
+PKG_CHECK_MODULES(VINO_TOOLS, glib-2.0 >= 2.12.0 gconf-2.0 gobject-2.0 >= 2.12.0 )
 
 dnl --enable-libnotify=(yes|no|auto)
 LIBNOTIFY_VERSION=0.4.4
Index: trunk/tools/Makefile.am
===================================================================
--- trunk/tools/Makefile.am	(revision 876)
+++ trunk/tools/Makefile.am	(working copy)
@@ -3,6 +3,7 @@
 INCLUDES = \
 	-DVINO_LOCALEDIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
 	$(VINO_TOOLS_CFLAGS) \
+	$(VINO_KEYRING_CFLAGS) \
 	$(WARN_CFLAGS) \
 	$(NULL)
 
@@ -14,5 +15,6 @@
 
 vino_passwd_LDADD = \
 	$(VINO_TOOLS_LIBS) \
+	$(VINO_KEYRING_LIBS) \
 	$(NULL)
 
