--- vino-2.26.0/configure.in~	2009-03-02 17:36:20.000000000 +0100
+++ vino-2.26.0/configure.in	2009-03-18 18:08:12.317341821 +0100
@@ -302,6 +302,6 @@
       getifaddrs (&myaddrs);
     ], have_getifaddrs=yes, have_getifaddrs=no)
-AM_CONDITIONAL(HAVE_GETIFADDRS, [test $"$have_getifaddrs" = "yes"])
+AM_CONDITIONAL(HAVE_GETIFADDRS, [test "$have_getifaddrs" = "yes"])
 
 dnl
 dnl End of libvncserver stuff
