diff -urN vino-2.10.0.orig/capplet/vino-preferences.desktop.in.in vino-2.10.0/capplet/vino-preferences.desktop.in.in
--- vino-2.10.0.orig/capplet/vino-preferences.desktop.in.in	2005-03-20 15:21:20.000000000 +0100
+++ vino-2.10.0/capplet/vino-preferences.desktop.in.in	2005-03-20 15:26:02.000000000 +0100
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=vino
