--- configure.in.orig	2009-05-18 17:59:03.000000000 +0200
+++ configure.in	2009-08-11 00:50:05.948280785 +0200
@@ -171,7 +171,7 @@
 	      [  --disable-gnutls=[no/yes] don't build VNC over SSL support [default=no]],,
 	      enable_gnutls=yes)
 if test "$enable_gnutls" = "yes"; then
-  AM_PATH_LIBGNUTLS(1.0.0,
+  PKG_CHECK_MODULES(LIBGNUTLS, gnutls >= 1.0.0,
 	            [AC_DEFINE(HAVE_GNUTLS, 1, [Defined if the GNU TLS library is present])],
                     AC_MSG_WARN([[
 ***
